pkgs/http/lib/http.dart

```diff
@@ -157,7 +166,8 @@ Future<String> read(Uri url, {Map<String, String>? headers}) =>
 ///
 /// For more fine-grained control over the request and response, use [Request]
 /// instead.
-Future<Uint8List> readBytes(Uri url, {Map<String, String>? headers}) =>
+Future<Uint8List> readBytes(Uri url,
```

Comment:
In addition, similar to here, I think it is completely possible to use Object? to set headers, because headers are made according to MDN standards, so in theory it can allow many structures to be converted into headers